src/components/Menu.js

```diff
@@ -1,21 +1,92 @@
+//Menu.js made by Jason Iqbal for SweatEquity, a segment of OilyHair.
+
 import React, {Component} from 'react';
 import '../css/Menu.css';
 import logo from '../img/Menu/se1.png';
 import selfie from '../img/selfie.jpg';
+import { AnonymousCredential } from 'mongodb-stitch-browser-sdk';
 
 class Menu extends Component {
-    // on button click change gamestate
+  
+    constructor(props) {
+        super(props);       
+        this.state = {
+            globalScores : [],
+            serverLoaded : false
+        };
+    }
+
     startGame = () => {
         this.props.setGameState('start');
     }
     
+    componentDidMount(){
+        let db = this.props.db;
+        this.props.client.auth.loginWithCredential(new AnonymousCredential()).then( user => {
+            let scoreCol = db.collection('sweatEquity');
+            scoreCol.find({}).asArray().then((allScores) => {
+                allScores.sort((a, b) => b.score - a.score);
+                this.setState({
+                    globalScores : allScores,
+                    serverLoaded : true
+                })
+            }).catch( error => { console.log(error); });  
+        });  
+    }
+    
+    ScoreList = (props) => {
+        const ScoreListItem = this.ScoreListItem;
+        
+        if(this.state.serverLoaded) {
+            return(
+                <table className="Highscore-table">
+                    <tbody>
+                        <ScoreListItem  props = {props.props[0]}/>
+                        <ScoreListItem  props = {props.props[1]}/>
+                        <ScoreListItem  props = {props.props[2]}/>
+                        <ScoreListItem  props = {props.props[3]}/>
+                        <ScoreListItem  props = {props.props[4]}/>
+                        <ScoreListItem  props = {props.props[5]}/>
+                        <ScoreListItem  props = {props.props[6]}/>
+                        <ScoreListItem  props = {props.props[7]}/>
+                        <ScoreListItem  props = {props.props[8]}/>
+                        <ScoreListItem  props = {props.props[9]}/>
+                    </tbody>  
+                </table>
+            )
+        }
+        return (
+          <p> Loading... </p>
+        )
+    }
+    
+    ScoreListItem = (data) => {
+        if(this.state.serverLoaded && data.props && data.props.score) {       
+            return(
+                <tr>
+                    <th>{data.props.name}</th>
```

Comment:
If you find yourself doing data.props like this, you've probably made a mistake. If this is a component on its own, you would just have it be props, no need for a `data` var.